docs/reference/cli/plugin.md

```diff
@@ -79,9 +89,9 @@ Update all plugins to the latest in the installed stream:
 steampipe plugin update --all
 ```
 
-Update the aws plugin to the latest in the 0.1 minor stream:
+Update the aws plugin to the latest version meeting the constraint:
 ```bash
-steampipe plugin update aws@0.1
+steampipe plugin update aws@^0.107
```

Comment:
I would expect `steampipe plugin update aws` to update to the latest version that meets the current constraint.  Does `steampipe plugin update aws@^0.107` actually change the constraint in the mod file as well?

Reply:
Plugin update requires specifying the installed version you wish to update (omitting it is same as `latest`)

For example, you can install 3x versions of `aws`:
- plugin install aws (this could be version 3.0.1)
- plugin install aws@^1.0 (this could be 1.0.0)
- plugin install aws@^2.0 (this could be 2.2.2)

plugin update requires you to target the installation you wish to update
- plugin update aws (3.0.1 -> 3.0.2)
- plugin update aws@^2.0 (2.2.2 -> 2.3.0)
- plugin update aws@^3.0 (error plugin not installed)